test/issues.jl

```diff
@@ -708,7 +708,15 @@
             f
         end
         """
-        @test fmt(str, always_use_return = true) == str
+        str_ret = """
+        function __init__()
+            return @doc raw\"""
+                   Doc string.
+                   \"""
+            f
+        end
+        """
+        @test fmt(str, always_use_return = true) == str_ret
```

Comment:
@domluna Your call on what this should do, but I think this is the more correct interpretation of `always_use_return` (and yes, it does the right thing).

Reply:
https://github.com/domluna/JuliaFormatter.jl/issues/405

did the parsing of this change with v3.2.4?

Reply:
Yes. I added a special case for doc parsing, so this parses `@doc "asd" f` as one expression, which is consistent with `Meta.parse`:
```
julia> str = "a"
"a"

julia> function f end
f (generic function with 0 methods)

julia> function __init__()
           @doc """
           Here's a docstring for `f` that includes dynamic content: $(str).
           """
           f
       end
__init__ (generic function with 1 method)

julia> __init__()
f

julia> function __init__()
           return @doc """
           Here's a docstring for `f` that includes dynamic content: $(str).
           """
           f
       end
__init__ (generic function with 1 method)

julia> __init__()
f
```